### PR TITLE
added the program of brackets as mentioned in the issue #21

### PR DESCRIPTION
**Pull request by VedRatan**
added the program in DSA directory

### CODE REVIEW

**Comment by Gauravsharma-20** [comment]
Add the program in Good-First-Issues folder and name the file Generate_Balanced_Parentheses.cpp

**Review by Gauravsharma-20 — APPROVED** [review]

**Comment by VedRatan** [comment]
done the job as specified sir

**Review by ultimatecoder2 — COMMENTED** [review]
Please add a case for n=0 too. Your program will go for infinite recursive calls. 
Rest seems fine.

**Comment by VedRatan** [comment]
> Please add a case for n=0 too. Your program will go for infinite recursive calls.
> Rest seems fine.

ohh ya got it thanks a lot, updating soon.

**Comment by VedRatan** [comment]
handled the egde case sir

**Review by ultimatecoder2 — APPROVED** [review]

**Comment by Gauravsharma-20** [comment]
@VedRatan please remove the two stack file we are just accepting 1 good first issue per user for now, we'll add more issues in the coming days.

File: Good-First-Issues/two_stack_using_single_array.cpp [hunk 1]
```diff
@@ -0,0 +1,131 @@
+#include <bits/stdc++.h>
+using namespace std;
+class twostack
+{
```
Comment (Gauravsharma-20) [inline Good-First-Issues/two_stack_using_single_array.cpp:4]: please remove this file

**Review by Gauravsharma-20 — CHANGES_REQUESTED** [review]

File: Good-First-Issues/two_stack_using_single_array.cpp (hunk 1 above)
Comment (VedRatan) [reply Good-First-Issues/two_stack_using_single_array.cpp:4]: okk sir

**Review by Gauravsharma-20 — APPROVED** [review]

**Review by Vibhu1710 — APPROVED** [review]

**Pull request merged**